docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+---
+title: "PIM Connector Framework"
+description: Import Data from any PIM
+---
+
+To upload huge amount of data to Unchained, for example if you pump the data from a PIM or ERP system, we have created a special API.
+
+## Bulk Import API
+
+Using the Bulk API stores the data in the Unchained Work Queue before processing and enables some neat features:
+
+1. Cloud Native: Allows dedicated background or worker instances to do the actual processing
+2. Transparent Process: Also stores the result on the work item, making successful and failed imports queryable.
+3. Captures Sync issues and reports those with a Status E-Mail to a central E-Mail Address (Sync error reporting)
+4. Performance: Only downloads assets that are not downloaded yet and tries to optimize the actual write calls to the database by using MongoDB bulk operations, making sync processing fast
+5. Push-based: Immediate representation of changes
+
+In some situations, it's propably wise to develop a sync microservice: You have a source system that
+- generates "Pull-based" data feeds, or
+- can not adopt to the JSON described below or
+- needs to merge data from different systems
+
+### Endpoint
+
+The main entry point to send import events is:
+Mutation.sendBulkImportEvents. The mutation takes an array of events that consist out of a type, an operation and a payload.
+
+Supported entity types:
+- PRODUCT
+- ASSORTMENT
+- FILTER
+
+Supported operations:
+- CREATE
+- UPDATE
+- REMOVE
+
+All Events follow this JSON Structure:
+
+```json
+{
+  "entity": "ENTITY TYPE",
+  "operation": "OPERATION TYPE",
+  "payload": { ... }
+}
+```
+
+Always try to send as many events at a time, so Unchained can optimize write operations.
+
+## JSON Reference
+
+### Entity Type: Product
+
+Set by unchained:
+- authorId,
+- slug history
+- _id, contentHash, created & updated if not provided
+
+```json
+{
+  "entity": "PRODUCT",
+  "operation": "CREATE",
+  "payload": {
+    "specification": {
+      "_id": "A",
+      "created": null,
+      "updated": null,
+      "sequence": "",
+      "tags": ["nice"],
+      "type": "SimpleProduct",
+      "status": "ACTIVE",
+      "published": "2020-01-01T00:00Z",
+      "commerce": {
+        "salesUnit": "ST",
+        "salesQuantityPerUnit": "1",
+        "pricing": [
+          {
+            "isTaxable": true,
+            "isNetPrice": true,
+            "countryCode": "CH",
+            "currencyCode": "CHF",
+            "amount": 10000,
+            "maxQuantity": null
+          }
+        ]
+      },
+      "warehousing": {
+        "baseUnit": "ST",
+        "sku": ""
+      },
+      "supply": {
+        "weightInGram": 0,
+        "heightInMillimeters": 0,
+        "lengthInMillimeters": 0,
+        "widthInMillimeters": 0
+      },
+      "proxy": {
```

Comment:
I do not understand what a proxy is by merely looking at this example. Is it explained elsewhere?

Reply:
Good catch. That is when a product "proxies" other products, which is the case with ConfigurableProducts and BundleProducts which when you add them to a cart, resolve to actual SKU's by some algorithm. That could also be the case for "Alias Products" or "Surrogate Products" we have discussed a while ago.

But currently, the BundleProduct uses the bundleItems data so it's maybe better to call this "variantResolvers" and skip proxy.assignments